### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by iLynette**
In this project I implemented the following;

- [ ] initialised React app.

- [ ] Create a directory for my re-usable components.

- [ ] Add `React Router `and set two `Routes` and `Link` for the app's navigation.

Books - the default view:

- [ ] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).

- [ ] Create a component called Book for displaying a single book (with a title and an author) and reuse it in a component that displays a list of books.

- [ ] Add a form for adding a book (no functionality yet).

- [ ] Created a separate component for this form (with inputs for a title and an author).

- [ ] Added categories:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+- HTML
+- CSS
+# Other tools
+- Babel compiler
+# Getting Started with Create React App
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## installation
+In order to run on your local machine, type in;
+
```
Comment (mikethreels) [inline README.md]: - [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

File: src/components/AddBook.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+//                   <option value="Action">Action</option>
+//                   <option value="fiction">Science Fiction</option>
+//                   <option value="Economy">Economy</option>
+//               </select>
+//               <input type="submit" value="ADD BOOK" className="buttn"></input>
+//           </form>
+//       </div>
+//     );
+//   }
+// }
```
Comment (mikethreels) [inline src/components/AddBook.js]: - [ ] please remove any commented out code because this will make your codebase look unprofessional and harder to read.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Lynette, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 2 above)
Comment (iLynette) [reply src/components/AddBook.js]: Thank you for the review @mikethreels i have resolved this

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @iLynette, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**